openghg_inversions/hbmcmc/inversion_pymc.py

```diff
@@ -523,27 +522,30 @@ def inferpymc_postprocessouts(
             scalemap_mode[bfds.values == (npm + 1)] = np.mean(xouts[:, npm])
 
     if rerun_file is not None:
-        # Note, at the moment fluxapriori in the output is the mean apriori flux over the
-        # inversion period and so will not be identical to the original a priori flux, if
-        # it varies over the inversion period
-        emissions_flux = np.expand_dims(rerun_file.fluxapriori.values, 2)
-    elif flux:
-        emissions_flux = flux.data.flux.values
+        flux_array_all = np.expand_dims(rerun_file.fluxapriori.values, 2)
     else:
         if emissions_name == None:
             raise NameError("Emissions name not provided.")
```

Comment:
```suggestion
            raise ValueError("Emissions name not provided.")
```
A `NameError` is for when a variable name is used, but that variable hasn't been defined. We should use `ValueError` here because the value `None` is causing the error.